lib/matplotlib/pyplot.py

```diff
@@ -1406,6 +1406,12 @@ def axis(*v, **kwargs):
     as kwargs selectively to alter just those limits without changing
     the others.
 
+      >>> axis('square')
+
+    changes the limit ranges (*xmax*-*xmin*) and (*ymax*-*ymin*) of
+    the *x* and *y* axes to be the same, and have the same scaling,
+    resulting in a square plot.
+
```

Comment:
The boilerplate generation is applied only to plotting functions.  It adds the`gca()`, the `hold` kwarg, and the `draw_if_interactive`.  There are many functions that are only in pyplot.  If I remember correctly, `axis` started out as pyplot-only but was added to Axes later.  Certainly `boilerplate.py` could be modified to handle more Axes methods, but I don't think there is any point in doing anything like that until various other major changes settle down.